roles/pgbench/templates/workload.yml.j2

```diff
@@ -41,7 +41,7 @@ spec:
                done;
              fi"
       restartPolicy: OnFailure
-{% if item.1.pin_node is defined and item.1.pin_node|length and item.1.pin_node is not sameas "" %}
+{% if item.1.pin_node is defined and item.1.pin_node %}
```

Comment:
`item.1.pin_node` covers both the empty and none case? 

Should we add a test to our CI to catch/cover this?

Reply:
Assuming item.1.pin_node covers both cases that's good. Otherwise would the bellow suffice? It would vet there is some length to the pin node, hence not being blank.

```
{% if item.1.pin_node is defined and item.1.pin_node|length > 0 %}
```

Reply:
> `item.1.pin_node` covers both the empty and none case?

Yes. Per the docs, you should also be able to cover the undefined case with simply '{% if item.1.pin_node %}`, but in testing it this failed for me. The if logic in this patch was tested against the "full" list of scenarios and passed, as documented in issue #144 .

> Should we add a test to our CI to catch/cover this?

I can't really think of how we would do this. Suggestions?

Reply:
If this change covers the list of scenarios you've documented in the issue, then I don't think we'll need another test in our CI.

Reply:
@aakarshg you don't think any of our other CRs will suffer from this exact same issue? I tend to think they will.